qemu/tests/cfg/boot_e1000e_with_cpu_flag.cfg

```diff
@@ -0,0 +1,8 @@
+- boot_e1000e_with_cpu_flag:
+    only q35
+    only e1000e
+    only Windows
+    type = boot_e1000e_with_cpu_flag
+    cpu_model_flags += "hv_vendor_id=KVMKVMKVM"
```

Comment:
Missing a comma before `hv_vendor_id`.

Reply:
I have add it, please help me review it again,thanks.